README.md

```diff
@@ -22,6 +22,7 @@ Please click on following link to see the more info about the specific topic
 * [Computer Vision](computer-vision)
 * [Control Theory](control-theory)
 * [Deep Learning](deep-learning)
+* [Embedded](embedded)
 * [Internet of Things-IoT](iot)
```

Comment:
I think it should be `Embedded Systems`

Reply:
it has to match the folder name, so `embedded-systems` specifically